app/src/main/java/com/example/tasky/screens/LoginScreen.kt

```diff
@@ -0,0 +1,131 @@
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.tasky.R
+import com.example.tasky.ui.components.TextButton
+import com.ramcosta.composedestinations.annotation.Destination
+
+@Destination(start = true)
+@Composable
+fun LoginScreen(){
+    Column(modifier =Modifier.fillMaxSize().background(Black)) {
+        Box(
+            modifier = Modifier
+                .fillMaxWidth()
+                .background(Color.Black)
+                .weight(.2f)
+            ,
+            contentAlignment = Alignment.TopCenter,
+        ) {
+            Column(modifier =  Modifier.align(
+                Alignment.Center).padding(top = 20.dp)) {
+                Text(
+                    text = stringResource(R.string.welcome_back),
+                    style = TextStyle(
+                        fontSize = 28.sp,
+                        lineHeight = 30.sp,
+                        fontFamily = FontFamily(Font(R.font.inter_bold)),
+                        fontWeight = FontWeight(700),
+                        color = Color(0xFFFFFFFF),
+                    )
+                )
+            }
+        }
+        Spacer(Modifier.padding(8.dp))
+        Box(
+            modifier = Modifier
+                .fillMaxWidth()
+                .clip(RoundedCornerShape(topStart = 30.dp, topEnd =  30.dp))
+                .background(White)
+                .weight(.8f)
+
+                ,
+            contentAlignment = Alignment.Center
+        ){
+            Column(Modifier.padding(horizontal = 12.dp).align(Alignment.TopCenter)) {
+                Spacer(Modifier.padding(24.dp))
+                TextField(
+                    value = "",
+                    onValueChange = {},
+                    placeholder = {Text(stringResource(R.string.email_address))},
+                    modifier = Modifier.fillMaxWidth(),
+                    colors = TextFieldDefaults.colors(
+                        focusedIndicatorColor = Color.Transparent,
+                        unfocusedIndicatorColor = Color.Transparent,
+                        cursorColor = MaterialTheme.colorScheme.onSurface.copy(alpha = 0.5f),
+                        focusedContainerColor =  Color(0xFFF2F3F7),
+                        unfocusedContainerColor =  Color(0xFFF2F3F7),
+                    ),
+                    shape = RoundedCornerShape(percent = 50)
+
+                )
+                Spacer(Modifier.padding(8.dp))
+                TextField(
+                    value = "",
+                    onValueChange = {},
+                    placeholder = {Text(stringResource(R.string.password))},
+                    trailingIcon = { Image(
+                        painter = painterResource(id = R.drawable.baseline_visibility_off_24),
+                        contentDescription = stringResource(id = R.string.hide_visibility)
+                    ) },
```

Comment:
`Image` is rather used for real photos. If you have a vector graphic like here, I'd recommend to use `Icon` instead. That also has the benefit that you can easily change the tint